invenio_cli/helpers/commands.py

```diff
@@ -243,13 +262,117 @@ def signal_handler(sig, frame):
             'docker/nginx/test.crt', '--key', 'docker/nginx/test.key'
         ], env=run_env)
 
-        click.secho('Development server + worker are running!', fg='green')
+        click.secho(
+            'Instance running!\nVisit https://localhost:5000', fg='green')
         server.wait()
 
 
 class ContainerizedCommands(object):
     """Containerized environment CLI commands."""
 
-    def __init__(self):
+    def __init__(self, cli_config, docker_helper):
         """Constructor."""
-        pass
+        self.cli_config = cli_config
+        self.docker_helper = docker_helper
+
+    def update_statics_and_assets(self, install):
+        """Update application containers' statics and assets."""
+        project_shortname = self.cli_config.get_project_shortname()
+
+        click.secho('Collecting statics and assets...', fg='green')
+        # Collect into statics/ folder
+        self.docker_helper.execute_cli_command(
+            project_shortname, 'invenio collect --verbose')
```

Comment:
minor: Do we need verbose here as part of the command?

Reply:
Nope, not in `containerize`. Removing!